src/pages/ReportDetailsPage.tsx

```diff
@@ -263,6 +264,21 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
     const shouldShowWriteCapability = !isMoneyRequestReport;
     const shouldShowMenuItem = shouldShowNotificationPref || shouldShowWriteCapability || (!!report?.visibility && report.chatType !== CONST.REPORT.CHAT_TYPE.INVOICE);
 
+    const isPayer = ReportUtils.isPayer(session, moneyRequestReport);
+    const isSettled = ReportUtils.isSettled(moneyRequestReport?.reportID ?? '-1');
```

Comment:
I wonder if we need this default really
```suggestion
    const isSettled = ReportUtils.isSettled(moneyRequestReport?.reportID);
```

Reply:
This might be obsolete but I wanted to add it as an extra percussion to make sure this doesn't cause a crash anytime in the app.